Remotion/Web/Core/UI/Controls/FormGridManager.cs

```diff
@@ -1675,16 +1675,16 @@ private void RegisterValidationErrors (FormGridRow dataRow)
         {
           ValidationError validationError = (ValidationError) validationErrorList[i]!;
           //  Get validation message
-          string validationMessage = validationError.ValidationMessage;
+          PlainTextString validationMessage = validationError.ValidationMessage;
           //  Get tool tip, tool tip is validation message
-          if (!string.IsNullOrEmpty (validationMessage))
+          if (!validationMessage.IsEmpty)
           {
             if (toolTip.Length > 0)
               toolTip.AppendLine();
-            toolTip.Append (validationMessage);
+            toolTip.Append (validationMessage.GetValue());
           }
         }
-        dataRow.ValidationMarker = CreateValidationMarker (toolTip.ToString());
+        dataRow.ValidationMarker = CreateValidationMarker ((PlainTextString) toolTip.ToString());
```

Comment:
shouldn't we use PlainTextString.CreateFromText for better consistency?

Also, I apparently misremmber: we also have a cast for string to PlainTextString? Is this actually helpful or could this create more confusion since there's two ways to do things?

Reply:
I added the factory method `CreateFromText` onto `PlainTextString` to make it more consistent with the `WebString`. Cast and factory method do the same thing so we could get rid of one of them or keep them.

I personally find it more natural to use a cast operator when you have a string and want to pass it to something that requires a `PlainTextString`. We could not do it with `WebString` since there were two options (plaintext or encoded). But I also understand if we want to use only one of the methods. Your choice.

Reply:
I agree with the cast in principle but specific case with PlainTextString, the symmetry to WebString is the preference in the design. We should keep the calls consistent in re-motion, but I don't "care" about whether we have the cast or not for downstream usage, unless we want to give explicit guidance here.